src/__test__/components/ContentWrapper.test.jsx

```diff
@@ -143,7 +147,39 @@ describe('ContentWrapper', () => {
     // Plots and Tables link is disabled
     expect(menus.at(3).props().disabled).toEqual(true);
   });
+  it('Links are enabled if the selected project is processed', async () => {
+    getBackendStatus.mockImplementation(() => () => ({
+      loading: false,
+      error: false,
+      status: {
+        gem2s: {
+          status: 'SUCCEEDED',
+        },
+        pipeline: {
+          status: 'SUCCEEDED',
+        },
+      },
+    }));
+
+    const wrapper = await mount(
+      <Provider store={store}>
+        <ContentWrapper backendStatus={{}}>
+          <></>
+        </ContentWrapper>
+      </Provider>,
+    );
+    await wrapper.update();
+    const sider = wrapper.find('Sider');
+    expect(sider.length).toEqual(1);
+    const menus = wrapper.find(Menu).children().find(Item);
+    const visibleMenuLength = menus.length / 2;
 
+    expect(visibleMenuLength).toEqual(4);
```

Comment:
Can we have a list containing the menu names, and then check for the length and the text of the menu too?